src/chigame/games/views.py

```diff
@@ -1,15 +1,24 @@
+from django.shortcuts import render, redirect, reverse
 from django.views.generic import DetailView, ListView
 
-from .models import Game
-
+from .models import *
 
 class GameListView(ListView):
     model = Game
     queryset = Game.objects.all()
     template_name = "games/game_list.html"
 
+def lobby_list(request):
+    lobbies = Lobby.objects.all()
+    context = {"object_list": lobbies}
+    return render(request, "games/lobby_list.html", context)
```

Comment:
Maybe worth writing as a class for consistency, maybe not. I'm good with this though.

Reply:
Either way (function or class) works.